modules/t/phenotypeAdaptor.t

```diff
@@ -57,12 +59,13 @@ ok($p && $p->name eq 'ACH', "fetch_by_description");
 $p->name('test');
 $p->description('test');
 delete $p->{dbID};
-
+delete $p->{stable_id};
```

Comment:
Suggest:
- a test be added to test storing a stable_id
- the store test use hide/restore so that test can be running when keeping test database intact

Reply:
The test phenotypeAdaptor.t cannot be re-run when keeping the test database intact due to Duplicate Entries.
Suggest using hide/restore to resolve this - either for this PR or as a separate ticket.